sql/mysql/convert.go

```diff
@@ -85,42 +85,42 @@ func (d *Driver) mustFormat(t schema.Type) string {
 	return s
 }
 
-// ConvertSchema converts a schemaspec.Schema into a schema.Schema.
-func (d *Driver) ConvertSchema(spec *schemaspec.Schema, tables []*schemaspec.Table) (*schema.Schema, error) {
-	return schemautil.ConvertSchema(spec, tables, d.ConvertTable)
+// Schema converts a sqlspec.Schema into a schema.Schema.
+func (d *Driver) Schema(spec *sqlspec.Schema, tables []*sqlspec.Table) (*schema.Schema, error) {
+	return specutil.Schema(spec, tables, d.Table)
 }
```

Comment:
I don't think `Driver.Schema` is a good name.

Reply:
What would you prefer?, @rotemtam ?